Ridhwana/dev.to
# Frontend for the Configurable Sidebar Navigation

**_https://github.com/forem/forem/pull/10662 needs to be merged first_**

## What type of PR is this? (check all applicable)


## Description

- Render the sidebar navigation area frontend based on the new NavigationLinks table.   
- Improve the UI for the admin to give them more hints on how to configure the Navigation section.

## Related Tickets & Documents
Previous related PR's:
- https://github.com/forem/forem/pull/9444
- https://github.com/forem/forem/pull/10662 

Issues:
- https://github.com/forem/forem/issues/4661
- https://github.com/forem/forem/issues/5475

## QA Instructions, Screenshots, Recordings

- Add, edit and delete any link from http://localhost:3000/admin/navigation_links and make sure that they reflect on the frontend. 
- Play with the position and display_only_when_signed_options and ensure that they react approriately

## Added tests?


## Added to documentation?


## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](gif_link)
